src/qibo/models/qcnn.py

```diff
@@ -54,6 +59,21 @@ def __init__(
         twoqubitansatz=None,
         copy_init_state=None,
     ):
+        """
+        Initializes the QuantumCNN object.
+
+        Args:
+            nqubits (int): The number of qubits in the QCNN.
+            nlayers (int): The number of layers in the QCNN.
+            nclasses (int, optional): The number of classes for the classification task. Defaults to 2.
+            params (np.ndarray, optional): The initial parameters for the QCNN. If None, random parameters are generated. Defaults to None.
+            twoqubitansatz (qibo.models.circuit.Circuit, optional): A two-qubit ansatz for the convolutional layers. If None, a default ansatz is used. Defaults to None.
+            copy_init_state (bool, optional): Whether to copy the initial state for each shot in the simulation. If None, the behavior depends on the backend. Defaults to None.
```

Comment:
Fine also with this notation, but if you want you can use the [`typing`] python module:

```python
from typing import Optional

def function(nqubits: int, ..., copy_init_state: Optional[bool] = None){
    ...
}
```